skimage/future/graph/graph_cut.py

```diff
@@ -75,7 +75,7 @@ def cut_threshold(labels, rag, thresh, in_place=True):
 
 
 def cut_normalized(labels, rag, thresh=0.001, num_cuts=10, in_place=True,
-                   max_edge=1.0):
+                   max_edge=1.0,  max_rec=float('Inf')):
```

Comment:
Can we make this an int ? Or do we need float because it allows the user to specify infinity